### PR TITLE
Ruofei/work

### PR DESCRIPTION
**Pull request by XRFXRF**
add excel to json controler

### CODE REVIEW

**Comment by XRFXRF** [comment]
add xlsx to json controler

File: app/controllers/xlsx_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class XlsxController < ApplicationController
+  def tojson
+
+    excel = Roo::Excelx.new('/home/felix/se/PhD-Application-Review-System/Dummy_data.xlsx')
```
Comment (tanaypatankar) [inline app/controllers/xlsx_controller.rb:4]: This needs to be changed to a relative path. For now you can probably try relative refrencing instead. Line `../../Dummy_data.xlsx`

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
-  resources :candidates
+  get 'xlsx/tojson'
+  get 'csv/tojson'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  resources :candidates
```
Comment (bmnz77) [inline config/routes.rb:6]: this looks fine

**Review by bmnz77 — APPROVED** [review]
reviewed routes.rb

**Review by tanaypatankar — APPROVED** [review]

**Pull request merged**